profile/README.md

```diff
@@ -0,0 +1,19 @@
+## Welcome to thoughtbot on GitHub
+
+[thoughtbot] is known worldwide as a leading [design], [development], and
+[product management] consultancy providing end-to-end services along the
+complete product development life cycle. thoughtbot validates, builds, scales,
+maintains, and supports web and mobile products.
+
+We are one of the leading design and development companies on a global scale.
+We use an agile, human-centered strategy for digital products in every stage
+from upfront research, through design and development, so you have one partner
+for your business each step of the way. Since 2003, we have worked to produce
+higher-quality products while improving team processes and have done so
+successfully for over [1,000 clients.][case studies]
```

Comment:
"thoughtbot is a global consultancy renown for our community contributions and client outcomes. We use an agile, human-centered strategy for web and mobile products at every stage of the product lifecycle. Since 2003, we have  produced high-quality products for over 1,000 clients. We are proud to leave every project with a stronger team and improved processes."

Reply:
I think it would be "renowned"?

Reply:
@cpytel yes! Thank you for the catch.